src/main/java/teammates/ui/controller/PageData.java

```diff
@@ -525,11 +539,20 @@ public String getInstructorStudentCommentClearPendingLink(String courseId) {
         return link;
     }
     
-    public String getInstructorFeedbackDeleteLink(String courseId, String feedbackSessionName, String nextURL) {
+    /**
+     * Retrieves the link to submit request to delete the session.
+     * @param courseId course ID
+     * @param feedbackSessionName the session name
+     * @param isHome true to indicate the request is from the home page
+     * @return the link to submit request to delete the session with return page link
+     */
+    public String getInstructorFeedbackDeleteLink(String courseId, String feedbackSessionName, boolean isHome) {
```

Comment:
oh because previously the buildBasicFsForm was passing an empty string inside. This is causing an error after I updated InstructorFeedbackDeleteAction to use nextUrl as nextUrl will give me something like `*nextUrl=user=text@example.com` 
Should I change it to take in String then pass in the correct String when calling instead?